src/borg/helpers/passphrase.py

```diff
@@ -141,8 +141,8 @@ def __repr__(self):
         return '<Passphrase "***hidden***">'
 
     def kdf(self, salt, iterations, length):
-        if os.environ.get("BORG_TESTONLY_MOCK_KDF") is not None:
-            return b'X' * length
+        if os.environ.get("BORG_TESTONLY_WEAKEN_KDF") is not None:
+            iterations = 1
```

Comment:
could the code from `yes` module be used here?

as it is, there is some danger that people set this to 0 or 1.

Reply:
if not:
```
os.environ.get("BORG_TESTONLY_WEAKEN_KDF", "0") == "1"
```

Reply:
> if not:
> 
> ```
> os.environ.get("BORG_TESTONLY_WEAKEN_KDF", "0") == "1"
> ```

Good idea! This have not occurred to me